packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -61,9 +61,38 @@ <h2>
                             </oui-select-picker-section>
                         </oui-select-picker>
                     </div>
+                    <div class="col-md-6 col-xl-4 my-3">
+                        <oui-select-picker
+                            data-name="missing"
+                            data-description="{{:: 'pci_projects_project_training_jobs_submit_select_image_preset_request' | translate }}"
+                            data-picture=""
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_select_image_preset_missing_framework' | translate }}"
+                            data-variant="light"
+                            data-values="[preset]"
+                            data-required
+                            data-model="$ctrl.job.image"
+                            data-on-change="$ctrl.generateName()"
+                        >
+                            <oui-select-picker-section
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{ $ctrl.communityUrl }}"
+                                >
+                                    <span>OVH AI Community</span>
```

Comment:
Do we need a translation for **OVH AI Community** ?

Reply:
From what I could see on the OVH COmmunity Forum there are only French ang English languages supported, and for both the name is OVH Community, that's the reason why I did not add it in the translations.